From c0f5d9943e2617c1fe7549742a184f59af6d205c Mon Sep 17 00:00:00 2001
From: Natanael Copa <ncopa@alpinelinux.org>
Date: Wed, 1 Feb 2012 20:18:00 +0000
Subject: [PATCH] make: link udevd with -lrt

sd-daemon.c uses mq_getattr() and should link with -lrt as the
man mq_getattr(3) states.

Signed-off-by: Natanael Copa <ncopa@alpinelinux.org>
---
 Makefile.am |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index e88a243..d5af409 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -249,7 +249,7 @@ src_udevd_SOURCES = \
 	src/sd-daemon.h \
 	src/sd-daemon.c
 src_udevd_CFLAGS = $(udev_common_CFLAGS)
-src_udevd_LDADD = $(udev_common_LDADD)
+src_udevd_LDADD = $(udev_common_LDADD) -lrt
 src_udevd_CPPFLAGS = $(udev_common_CPPFLAGS)
 
 src_udevadm_SOURCES = \
-- 
1.7.9

